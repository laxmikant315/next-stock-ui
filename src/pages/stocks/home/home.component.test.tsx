import React from 'react';
import { shallow } from 'enzyme';
import Home from './home.component';

it('render test',()=>{

  expect(shallow(<Home/>)).toMatchSnapshot()
})
