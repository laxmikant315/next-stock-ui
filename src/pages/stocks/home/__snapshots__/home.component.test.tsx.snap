// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render test 1`] = `
<styled.div>
  <Tabs
    defaultActiveKey="1"
    hideAdd={false}
    tabPosition="top"
  >
    <TabPane
      key="1"
      placeholder={null}
      tab="Intraday"
    >
      <IntradayStocks
        liveData={Array []}
      />
    </TabPane>
    <TabPane
      key="2"
      placeholder={null}
      tab="Swing"
    >
      <lazy
        liveData={Array []}
        type="swing"
      />
    </TabPane>
  </Tabs>
</styled.div>
`;
